lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb

```diff
@@ -138,10 +138,9 @@ def support_files_temp_dir
           #
           # @return [void]
           #
-          def create_info_plist_file
-            path = target.info_plist_path
+          def create_info_plist_file(path, native_target, version, platform, bundle_package_type = :fmwk)
```

Comment:
NIT: I'd just remove the default parameter for `bundle_package_type ` and force it to be explicit to have less optional parameters. I think it makes understanding callers more clear IMO.